text/0000-deprecate-reset-namespace.md

```diff
@@ -0,0 +1,45 @@
+- Start Date: 2017/04/18
+- RFC PR: (leave this empty)
+- Ember Issue: (leave this empty)
+
+# Summary
+
+This RFC suggest the deprecation and eventual removal of the `resetNamespace` option
+in routes.
+
+# Motivation
+
+This little known feature is mostly used to have shorter route names on nested routes.
+It can be used to move a route hierarchy to a more nested location without having to rename
+all places where that route was referenced by its name (mostly `{{link-to}}`, but not exclusively).
+
+However, reseting the namespace of a route prevents developers to intuitively understand
+the placement of a route in the nesting hierarchy of the app by simply reading its name,
```

Comment:
Wouldn't it be easier to read by having `resetNamespace` on by default?

What I mean is, this way `this.route('deeply.nested.route')` will always be an absolute path rather than a relative path to the parent route.